Assets/Scripts/View/PlayerCanvas/FabricField.cs

```diff
@@ -5,14 +5,15 @@
 
 public class FabricField : MonoBehaviour
 {
-    Player player;
+    PlayerControl player;
```

Comment:
I don't have a proposal or solution, but it looks suspicious that several View components (I counted 4 so far) must have a PlayerControl.

But at least now I see what you meant when you wished we wouldn't have the Controller initially. Maybe it's not an ideal pattern here. But I'm not sure what would be better.